src/components/contextual/pages/vebal/LMVoting/LMVoting.vue

```diff
@@ -136,6 +134,45 @@ function handleModalClose() {
 function handleVoteSuccess() {
   refetchVotingGauges();
 }
+
+const intersectionSentinel = ref<HTMLDivElement | null>(null);
+const showingGaugesIdx = ref(10);
```

Comment:
This name could maybe be a bit clearer, like `renderedRows`

Reply:
fixed